codec-redis/src/main/java/io/netty/handler/codec/redis/AbstractStringRedisMessage.java

```diff
@@ -38,4 +39,14 @@
     public final String content() {
         return content;
     }
+
+    @Override
+    public String toString() {
+        return new StringBuilder(StringUtil.simpleClassName(this))
```

Comment:
Just use normal string concats or at least size the StringBuilder correctly ?

Reply:
As you can [see](https://github.com/netty/netty/pull/7687/commits/a9f2e7686aa5c9d64661554055830fbad17ae935#diff-c1537befe2d2bd6dd21162a189fee058L37) I just pulled up the `toString()` method from existing sub-classes. The implementation is similar to that of [other (core) classes](https://github.com/netty/netty/blob/73f23c5faa967748bdda826278ac5e63f715ae09/buffer/src/main/java/io/netty/buffer/AbstractByteBuf.java#L1332). Is a more complex implementation that calculates the needed size really worth it? I suggest to never call the `toString()` method in a productive environment anyway.

Reply:
I think it worth it... that said I think we can improve it in another PR.

Reply:
Great! I'd be happy to help with that :-) Anything else that should be done?